zowe/zlux-server-framework
# Get full list of discovery services using ZWE_DISCOVERY_SERVICES_LIST

# Provide full list of discovery services to Eureka JS Client

## Proposed changes
So far app-server communicated with a single discovery service. In case of failure of the discovery service, app server couldn't send heartbeat to another discovery service.  This PR configures Eureka JS Client in app-server with multiple discovery services, and in case of failure of one discovery service, app-server should be able to send heartbeat to one another. 

This PR addresses Issue: zowe/zlux#674

This PR depends upon the following PRs: 
* https://github.com/zowe/zlux-server-framework/pull/314

## Type of change
Please delete options that are not relevant.
- [ ] Change in a documentation
- [ ] Refactor the code 
- [ ] Chore, repository cleanup, updates the dependencies.

## PR Checklist

## Testing

* Configure HA instances with at least two Discovery Services enabled
* Run al the HA instance and wait until app-server registers in one of Discovery Services
* Stop the Discovery Service
* Ensure that app-server continues to send heartbeat to one of remaining Discovery Services
